src/pages/projectCreate/components/ProjectFeeSelection.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, HStack, Text, useRadio, useRadioGroup, VStack } from '@chakra-ui/react'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+type ProjectFeeSelectionProps = {
+  readOnly?: boolean
+  value: number
+  onChange: (value: number) => void
+}
+
+const ProjectFeeSelection: React.FC<ProjectFeeSelectionProps> = ({ readOnly, value, onChange }) => {
+  const { t } = useTranslation()
+
+  const options = [
+    { label: '0%', value: '0.0' },
+    { label: '2%', value: '0.02' },
+    { label: '4%', value: '0.04' },
+    { label: '6%', value: '0.06' },
+    { label: '8%', value: '0.08' },
+  ]
+
+  const { getRootProps, getRadioProps } = useRadioGroup({
+    name: 'transactionFee',
+    defaultValue: String(value),
+    onChange: (value) => onChange(Number(value)),
+  })
+
+  const group = getRootProps()
+
+  return (
+    <VStack
+      width="100%"
+      border="2px solid"
+      borderColor="neutral.200"
+      backgroundColor="primary.50"
+      borderRadius="8px"
+      alignItems="flex-start"
+      padding="20px"
+      spacing="20px"
+    >
+      <Text variant="body1" fontWeight="500">
+        {t('Support Geyser')}
+      </Text>
```

Comment:
Suggestion, you can simply use <Body1  semiBold> in this scenario. And a few others.

Reply:
I am not sure I understand. Use what?

Reply:
I had to put the ticks for github to show it.